app/javascript/components/rooms/room/Room.jsx

```diff
@@ -85,18 +98,50 @@ export default function Room() {
               }
           </Col>
           <Col>
-            <Button variant="brand" className="start-meeting-btn mt-1 mx-2 float-end" onClick={startMeeting.mutate} disabled={startMeeting.isLoading}>
-              {startMeeting.isLoading && <Spinner className="me-2" />}
-              { room?.online ? (
-                t('room.meeting.join_meeting')
-              ) : (
-                t('room.meeting.start_meeting')
-              )}
-            </Button>
-            <Button variant="brand-outline" className="mt-1 mx-2 float-end" onClick={() => copyInvite()}>
-              <Square2StackIcon className="hi-s me-1" />
-              { t('copy') }
-            </Button>
+            <Row>
+              <Col className="col-12">
```

Comment:
Is the extra row and column needed here? Considering it's already wrapped in a column

Reply:
Yeah without it looks like this. screenshot added.

![image](https://github.com/bigbluebutton/greenlight/assets/28535207/6f885cb6-e4ae-49f2-a60a-3d362b2c3e07)

Reply:
Try using `Stack` instead

Reply:
I tried with a Stack but it still was same as screenshot above...tried with a Stack inside the Row but same again